inc/core/templates/theme-notification-banner.php

```diff
@@ -0,0 +1,315 @@
+					<h3 id="active-pattern">Notification Settings</h3>
+					
+					<div class="field-grid">
+						<div class="field">
+							<div class="label">
+								<label for="notice_enabled">Enable Admin Notification</label>
+							</div>
+							<div class="input">
+								<div class="true-false">
+									<label>
+										<input 	type="checkbox" 
+												id="notice_enabled" 
+												name="notice_enabled" 
+												value="1" 
+												<?php
+                                                if ( $notice_enabled ) {
+													echo ' checked="checked"';
+												}
+                                                ?>
+												/>
+										<span class="message">Enable / disable banner</span>
+									</label>
+								</div>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_homepage_only">Homepage only</label>
+							</div>
+							<div class="input">
+								<div class="true-false">
+									<label>
+											<input 	type="checkbox" 
+													id="notice_homepage_only" 				
+													name="notice_homepage_only" 
+													value="1" 
+													<?php
+                                                    if ( $notice_homepage_only ) {
+														echo ' checked="checked"';
+													}
+                                                    ?>
+													/>
+											<span class="message">Show only on the homepage</span>
+									</label>
+								</div>
+							</div>
+						</div>
+						<div class="field">
+							<div class="label">
+								<label for="notice_status">Status (colour)</label>
+							</div>
+							<div class="input">
+								<select id="notice_status" class="" name="notice_status">
+									<option value="notice-default" 
+									<?php
+									if ( 'notice-default' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Default (BCGov blue)</option>
+									<option value="notice-green" 
+									<?php
+									if ( 'notice-green' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Notice (green)</option>
+									<option value="notice-white" 
+									<?php
+									if ( 'notice-white' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Minimalist (white)</option>
+									<option value="notice-black" 
+									<?php
+									if ( 'notice-black' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Cancelled (dark)</option>
+									<option value="notice-yellow" 
+									<?php
+									if ( 'notice-yellow' === $notice_status ) {
+											  echo ' selected';
+									}
+									?>
+                                            >Warning (yellow)</option>
```

Comment:
I know this is for a backend option but does the prefixed greater than symbol harm screenreader comprehension? If this theme ends up being used widely then it could be something to consider.

Reply:
Argh. The php linting is going to be the death of me. This is not how the code was implemented.